docs/references/uvh5_memo.tex

```diff
@@ -269,19 +306,35 @@ \subsection{Visdata Dataset}
 datatype. For examples of how to handle complex integer datatypes in
 \verb+h5py+, see Appendix~\ref{appendix:integers}.
 
+\subsubsection{Conjugation Convention}
+A cross-correlation between two antennas is defined by the baseline connecting
+them, and the conjugation of one of the input data streams. Accordingly, the
+\textit{uvw} coordinates and the conjugation of the visibility data are
+interconnected, based on the definition of one's coordinate system. For UVH5
+files, it is assumed that the convention for the Radio Interferometer
+Measurement Equation (RIME) of a visibility $\mathcal{V}$ for antennas $i$ and
+$j$ is as follows:
+\begin{equation}
+\mathcal{V}(u_j - u_i, v_j - v_i) = \int \dd{l} \dd{m} I(l, m) g_i(l, m) e^{-2\pi i (u_i l + v_i m)} g_j^*(l, m) e^{2\pi i (u_j l + v_j m)}.
+\end{equation}
+That is, the baseline vector defined by the $uvw$ coordinates is directed from
+antenna $i$ to antenna $j$ (so the baseline vector can be computed as
+$\vb{r_j} - \vb{r_i}$, where $\vb{r}$ is the position vector of a given
+antennas), and the data corresponding to antenna $j$ is conjugated. Note that if
+a file is generated with the opposite convention, it is usually sufficient to
+multiply $uvw$ coordinates by $-1$ to generate a self-consistent dataset.
```

Comment:
I think you'd want to conjugate the data as well when you do this. At least that's true if the data are matched to the other convention.